packages/babel-parser/test/fixtures/core/categorized/invalid-object-setter-no-param/output.json

```diff
@@ -0,0 +1,52 @@
+{
```

Comment:
This is also tested in https://github.com/babel/babel/blob/18d13d0032825b6d690b81f9d7e1d5a22ec97b30/packages/babel-parser/test/fixtures/core/object/invalid-setter/input.js

Reply:
Good catch! I will check if the other accessor params test already have duplicates, if so I tend to just remove them.